### DISCUSSION TITLE
记 Github 学生认证 - 愧怍的小站

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#168] [General]
# 记 Github 学生认证 - 愧怍的小站

我个人是非常讨厌这些认证提交手续的，例如疫情健康报告，请假申请表等等，当然也包括这次 Github 学生认证。

https://kuizuo.cn/github-student-certification#comments

**Comment by inannan423** [comment]
连校园网秒过😎

**Reply by kuizuo** [reply]
我同学基本都是秒过的，就我比较黑，认证了11次才过😂

**Comment by lumsyte** [comment]
谢谢博主,帮我找到了学生包灵感(我搜索了好久,就你这个靠谱),我通过了🤣

**Reply by kuizuo** [reply]
通过就好。平常记得多刷刷github，可以了解到很多新技术以及感兴趣的项目。

**Comment by nowhereman1999** [comment]
你好,大佬,请问您文中提到的"修改 host，然后需要 Google 地图的时候开启科学上网，然后获取定位信息后再关闭，最后提交的时候没开启科学上网"这一步具体怎样操作呢,我现在因为在家,必须得开梯子才能够访问github,但是后面上传证明的时候只能够拍摄上传,所以想问问大佬这里具体怎么操作的

**Reply by kuizuo** [reply]
修改 host 的操作是为了绕过DNS解析，相当于直接访问 github.com 的ip地址。关于这个操作你可以参阅 [GitHub无法访问、443 Operation timed out的解决办法](https://juejin.cn/post/6844904193170341896)

因为 Google 地图必须开启科学上网，所以你可以在获取到定位信息后（前几步的时候开启科学上网），最后（建议）将科学上网关闭，然后提交表单，等待学生认证结果。

最后你现在在家而不在学校的话，真实定位上有可能会和你学校地址不符，也有可能无法认证成功。这是你需要注意的一点，因为当时我暑假在家就是这样无法认证。

最后也是希望你能尽早认证成功，认证完就能白嫖copilot，jetbrains，AWS 服务器等等。只要坚持不懈一定能够认证成功了💪！

**Reply by nowhereman1999** [reply]
没想到大佬这么快就回复我了，我现在也打算还是先等回学校再申请了，在家里确实始终有各种问题，不过最后还是谢谢大佬详尽的解答了！

**Reply by nowhereman1999** [reply]
> 修改 host 的操作是为了绕过DNS解析，相当于直接访问 github.com 的ip地址。关于这个操作你可以参阅 [GitHub无法访问、443 Operation timed out的解决办法](https://juejin.cn/post/6844904193170341896)
> 
> 因为 Google 地图必须开启科学上网，所以你可以在获取到定位信息后（前几步的时候开启科学上网），最后（建议）将科学上网关闭，然后提交表单，等待学生认证结果。
> 
> 最后你现在在家而不在学校的话，真实定位上有可能会和你学校地址不符，也有可能无法认证成功。这是你需要注意的一点，因为当时我暑假在家就是这样无法认证。
> 
> 最后也是希望你能尽早认证成功，认证完就能白嫖copilot，jetbrains，AWS 服务器等等。只要坚持不懈一定能够认证成功了💪！

大佬，非常激动告诉你我今天申请学生认证成功了，而且竟然还是秒过，我刚刚一开始直接上传学信网证明一直提示需要，后来我把翻译后的原件和原件打印出来后用手机拍照在上传就过了，现在已经开始用上copilot了，非常感谢你之前详细的解答！

**Comment by JasonLou99** [comment]
认证失败了好几次，然后突然好像就没提示失败了，显示：We have already received a discount request for @xxxx. We will get back to you soon，话说这要等多久

**Reply by kuizuo** [reply]
这个我也说不准，至少得到了不一样的答复，成功的概率还是有的。

**Comment by xycld** [comment]
我也11次了，不过还没成功，不过我上传的是录取通知书，也许是这个的问题吧，手机上不去我就用的OBS照，以及我没改名

**Reply by xycld** [reply]
现在交都不让我交了QAQ

**Comment by out-of-fool** [comment]
为什么我打不开学生认证的页面
